test-requirements.txt

```diff
@@ -1,5 +1,3 @@
--r torchvision-requirements.txt
-
```

Comment:
Dont we still need it for tests ?

Reply:
I moved it to the top level requirements file, because we also need these other packages here for the stable build. Do you have another idea how to do it?

Reply:
Ok, I will just call the file directly from` build_linux_packages.sh`, that should remove the dependency

Reply:
This sounds good but let's also make sure the regular CI doesn't require a similar direct install